src/components/views/elements/DeviceVerifyButtons.js

```diff
@@ -65,7 +65,8 @@ export default React.createClass({
                         <ul>
                             <li><label>Device name:</label> <span>{ this.state.device.getDisplayName() }</span></li>
                             <li><label>Device ID:</label>   <span><code>{ this.state.device.deviceId}</code></span></li>
-                            <li><label>Device key:</label>  <span><code><b>{ this.state.device.getFingerprint() }</b></code></span></li>
+                            <li><label>Device fingerprint:</label>  <span><code><b>{ this.state.device.getFingerprint() }</b></code></span></li>
```

Comment:
Makes sense to keep it separate. Sorry for causing conflicts.

I agree that there is a bit of a naming confusion going on at the moment. "Signing key" and "encryption key" might be better terms, though tbh in the UI I would prefer the user to be able to forget there is more than one key.